src/Evolution/VariableFixing/FixToAtmosphere.hpp

```diff
@@ -36,9 +36,10 @@ namespace VariableFixing {
 /// \ingroup VariableFixingGroup
 /// \brief Fix the primitive variables to an atmosphere in low density regions
 ///
-/// If the rest mass density is below the specified value, it is
-/// raised to the specified value, and the pressure, specific internal
-/// energy (for one-dimensional equations of state), and specific
+/// If the rest mass density is below the specified value,
+/// \f$\rho_{\textrm{cutoff}}\f$ (DensityCutoff), it is raised to the
+/// \f$\rho_{\textrm{atm}}\f$ (DensityOfAtmosphere), and the pressure, specific
```

Comment:
Re: what this is for --- I discussed this in slack yesterday with Nils and Larry. The idea (from here https://arxiv.org/abs/1306.4953) is that you want to make sure your atmosphere stays as an atmosphere. The previous version of the code would set smaller values to a floor, but if the floor evolved to higher density there was no longer any control of the fluid state. This version sets rho to rho_atm if rho < rho_cutoff with rho_cutoff >= rho_atm, and this means you have a buffer (between rho_atm and rho_cutoff) where the atmosphere can evolve but still be recontrolled back to atmosphere. To go from atmosphere to "real" densities, you need to jump from rho_atm to rho > rho_cutoff in one substep, but for sane choices of these parameters this easily happens near resolved meaningful matter flows. I found this fixing approach helpful in my TOV experiments.

It is true that the comment is misleading, because if rho < rho_cutoff, the density could be raised _or_ lowered to rho_atm. Best to say "it is set to" I think.